modules/nodes-custom-scheduler-pods.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+//
+// * nodes/scheduling/nodes-custom-scheduler.adoc
+
+[id="nodes-custom-scheduler-deploying_{context}"]
+= Deploying Pods using a custom scheduler
+
+After the custom scheduler is deployed in your cluster, you can configure Pods to use that scheduler instead of the default scheduler.
+
+.Procedure
+
+. Create a Pod configuration and specify the name of the custom scheduler in the `schedulerName` parameter.
```

Comment:
Before this step there is one more steps which needs to be performed. Below are the details

we need the content from "If RBAC is enabled on your cluster" under "Enable leader Election" here https://kubernetes.io/docs/tasks/extend-kubernetes/configure-multiple-schedulers/ with out which we cannot get any pod using custom-scheduler.